api/routes/trackers.py

```diff
@@ -0,0 +1,41 @@
+from fastapi import APIRouter
+import random
+
+from models.trackers import Tracker, trackers
+
+router = APIRouter()
+
+
+@router.get("/")
+def read_trackers_all():
+    return trackers
+
+
+@router.get("/{tracker_id}")
+def read_tracker_by_id(tracker_id: int):
+    if tracker_id in trackers.keys():
+        return trackers[tracker_id]
+    return {"message": "Tracker not found"}
+
+
+@router.post("/")
+def create_tracker(tracker: Tracker):
+    tracker_id = random.randint(1, 9999)
+    trackers[tracker_id] = tracker
```

Comment:
before we assign, we should check whether there is no tracker with the same randomly generated id